Gemfile

```diff
@@ -3,6 +3,8 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '3.0.0'
 
+# use cancancan
```

Comment:
- [ ] Make sure you have rswag gem installed. 
```
gem 'rswag-api'
gem 'rswag-ui'

```
```
group :development, :test do ... 
gem 'rswag-specs'
 end 
```

Reply:
Hi @arslanbisharat 
Please I would like to know why you asked us to install this gems. I believe they are part of the milestone requirement

Reply:
Hi @arslanbisharat
Although we installed them

Reply:
Hello, you can remove those gems, we don't need it at this moment.

Reply:
I will remove the gems again